apps/editing-toolkit/editing-toolkit-plugin/wpcom-universal-themes/index.php

```diff
@@ -26,8 +26,20 @@ function is_core_fse_active() {
 		return false;
 	}
 
-	// Now we just check for our own option.
-	return (bool) get_option( ACTIVATE_FSE_OPTION_NAME );
+	$universal_theme_template = defined( 'IS_ATOMIC' ) && IS_ATOMIC
+		? 'blockbase'
+		: 'pub/blockbase';
```

Comment:
Will Blockbase always be the default template for universal themes? Im guessing that is the assumption here, but maybe a comment would be good to clarify.

Should we consider instead doing this check by theme tag and have separate theme tags to distinguish between universal vs. pure-FSE block themes?

Reply:
> but maybe a comment would be good to clarify.

This makes sense to me 👍 I'll add one in.

> Should we consider instead doing this check by theme tag and have separate theme tags to distinguish between universal vs. pure-FSE block themes?

I like this suggestion because pure-FSE block themes are, in fact, functionally different from universal themes.

The easiest way I can envision doing this is simply adding an some sort of additional theme tag like `block-only` while leaving the existing usage of `block-templates` tags untouched. For example, universal themes would continue to use the `block-templates` tag, while block themes would use both the `block-templates` and `block-only` tag. This way, existing references to `block-templates` would continue to behave as expected. 

This, however, might be confusing from the perspective of newer developers. 🤔 Either way, curious about others' thoughts @mattwiebe @creativecoder

Reply:
> adding some sort of additional theme tag like block-only while leaving the existing usage of block-templates tags untouched

I don't think this will work well, there will be third party "block-only" themes (pNEWy-exy-p2). We won't control the tags for those themes on dotorg. We could modify the tags on wpcom when those themes are copied over for Simple sites, but than we would have to account for the difference with Atomic sites using the dotorg version.

To add to that, the dual source of tags (server-side they use the style.css header comment, but in Calypso they use the theme.wordpress.com post tags), make them a sometimes inconsistent source of truth.

I think using a check for blockbase as the parent theme will work just fine, for now.